Languages/german.lang

```diff
@@ -598,13 +599,13 @@ msgid "Default Settings"
 msgstr "Einstellungen zurücksetzen"
 
 msgid "Deflicker Filter"
-msgstr ""
+msgstr "Deflicker-Filter"
```

Comment:
` Deflicker`  is not German.

Reply:
There is no proper translation for 'Deflicker Filter' as 'Deflicker' is a proper noun. According to DIN 5008, a hyphen can be used to combine those two words. But let's not get too picky about it. If you find a better translation, please let me know.